R/AbstractAnnData.R

```diff
@@ -68,6 +68,14 @@ AbstractAnnData <- R6::R6Class("AbstractAnnData",
     n_vars = function() {
       nrow(self$var)
     },
+    #' @description Keys ('column names') of `obs`.
+    obs_keys = function() {
+      .abstract_function()
+    },
+    #' @description Keys ('column names') of `var`.
+    var_keys = function() {
+      .abstract_function()
+    },
```

Comment:
Can we not implement this in the abstract class?

```suggestion
    obs_keys = function() {
      names(self$obs)
    },
    #' @description Keys ('column names') of `var`.
    var_keys = function() {
      names(self$var)
    },
```

Reply:
Is it guaranteed (or 'expected') that self$var returns something with `names()`?

Reply:
Indeed :)